src/plugins/kibana_react/public/util/react_mount.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { I18nProvider } from '@kbn/i18n/react';
+import { MountPoint } from 'kibana/public';
+
+/**
+ * Mount converter for react components.
+ *
+ * @param component to get a mount for
+ */
+export const reactMount = (component: React.ReactNode): MountPoint => {
```

Comment:
I'd like to see this renamed if possible to be a little clearer and reduce redundancy. This is part of the `kibana_react` package, so I think it's safe to leave `react` out of the name. Maybe something like `toMount` or `toMountPoint`? I'd also like to avoid using *just* the word `mount` here as it implies an immediate mount, which isn't the case, rather just thunking the rendering.

In addition, idiomatic React typically reserves the word `component` for a proper component class, and since this is a node and *not* necessarily a component instance, I think the argument should reflect that.

```ts
export const toMountPoint = (node: React.ReactNode): MountPoint => {
  // ...
};
```

Reply:
Very valid point.